experimenter/experimenter/nimbus_ui_new/static/js/index.js

```diff
@@ -70,4 +70,14 @@ $(() => {
     setupTooltips();
     setupToasts();
   });
+
+  let isInternalChange = false;
+  $(".selectpicker").on("changed.bs.select", function () {
+    if (isInternalChange) {
+      return;
+    }
+    isInternalChange = true;
+    this.dispatchEvent(new Event("change", { bubbles: true }));
+    isInternalChange = false;
+  });
```

Comment:
Please add a comment explaining why we're doing this because it is not immediately clear to me why this is necessary.